lir/_plotting_new.py

```diff
@@ -261,16 +261,14 @@ def score_distribution(scores, y, bins=20, weighted=True, ax=plt):
     """
     ax.rcParams.update({'font.size': 15})
     bins = np.histogram_bin_edges(scores[np.isfinite(scores)], bins=bins)
+    bin_width = bins[1] - bins[0]
 
+    # flip Y-classes to achieve blue bars for H1-true and orange for H2-true
+    y_classes = np.flip(np.unique(y))
     # create weights vector so y-axis is between 0-1
-    scores_by_class = [scores[y == cls] for cls in np.unique(y)]
+    scores_by_class = [scores[y == cls] for cls in y_classes]
     weights = [np.ones_like(data) / len(data) for data in scores_by_class]
```

Comment:
Dit gaat mis indien `weighted is False`
```suggestion
    if weighted:
        weights = [np.ones_like(data) / len(data) for data in scores_by_class]
    else:
        weights = [np.ones_like(data) for data in scores_by_class]
```